package.json

```diff
@@ -28,8 +28,7 @@
     "build:es": "cross-env NODE_ENV=production BABEL_ENV=es babel ./src --out-dir ./dist/es --ignore *.test.js",
     "build": "yarn build:es2015 && yarn build:es2015modules && yarn build:es && yarn build:copy-files && yarn build:copy-flow-files",
     "unit-test": "yarn jest --coverage",
-    "localtunnel": "localtunnel",
-    "e2e": "node ./e2e/e2e.js",
+    "localtunnel": "lt",
```

Comment:
what does this do?

Reply:
it just creates a shortcut to the localtunnel package (so you don't have to do `./node_modules/lt`, but just `yarn localtunnel`), which exposes the web app externally